src/amber/cli/recipes/recipe.cr

```diff
@@ -40,6 +40,7 @@ module Amber::Recipes
         exit 1
       end
 
+      @name_plural = Inflector.pluralize(word)
```

Comment:
Where does `word` come from? Isn't this pluralizing `@name`?

Also, any reason not to create a getter, or write a custom one like this:
```crystal
def plural_name
  @plural_name ||= Inflector.pluralize(word)
end
```

Reply:
Hi @marksiemers :tada: 

You're completely right, let me fix that :100:

Reply:
> Also, any reason not to create a getter, or write a custom one like this:

Is a getter really needed? I think `initialize` method already does the job, no?

`@name_plural` is initialized just one time inside the constructor, so no memoization requirement

Reply:
It can be done without the getter, assuming it will never be needed publicly.

If it will ever be needed publicly, then creating a private getter now will make things more consistent when that change comes.

I know this isn't ruby, but one of the ruby idioms that I follow is that you minimize directly accessing instance variables (ideally only in the constructor/initialize method).

Reply:
@marksiemers Yeah, nice suggestion :+1:

I think we can refactor this in a new PR (`@name_plural` and others)  :sweat_smile:

Reply:
BTW, nice to see you here again :smile: :+1: